client/settings/transactions-and-deposits/test/index.test.js

```diff
@@ -28,6 +30,7 @@ describe( 'TransactionsAndDeposits', () => {
 		useManualCapture.mockReturnValue( [ false, jest.fn() ] );
 		useGetSavingError.mockReturnValue( null );
 		useSavedCards.mockReturnValue( [ false, jest.fn() ] );
+		useCardPresentEligible.mockReturnValue( [ false ] );
```

Comment:
It'll help to also test a scenario where `useCardPresentEligible` returns true. The message displayed will vary in both cases. Your expectation will be similar to what we see here:

https://github.com/Automattic/woocommerce-payments/blob/develop/client/settings/transactions-and-deposits/test/index.test.js#L97-L101

Reply:
just added some tests.